src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -12,4 +12,5 @@ public Bulldozer() {
     public void doWork() {
         System.out.println("Bulldozer started to work");
     }
+
```

Comment:
The task description requires adding custom fields that are specific to a bulldozer. Consider adding fields such as `bladeWidth`, `enginePower`, or any other attributes that are unique to a bulldozer.